src/EdjCase.JsonRpc.Router/Abstractions/IRpcMethodProvider.cs

```diff
@@ -27,11 +27,11 @@ public RpcRouteMetaData(IReadOnlyList<IRpcMethodInfo> baseMethods, IReadOnlyDict
 	public interface IRpcMethodInfo
 	{
 		string Name { get; }
+		MethodInfo SourceMethodInfo { get; }
```

Comment:
This won't work either. I have abstracted out the MethodInfo and am using `IRpcMethodInfo` in its place. If we want this we need to have some custom object/string to represent what the declaring type is